# Please DO NOT edit this file
# Make a copy instead, make changes and pass the absolute path to the copy in arguments
general:
  publish_interval: 30  # time between two published messages
comstation:
  enable: true
  port: "/dev/ttyUSB0"  # COM port of the device
  work_period: 300      # time between two measurements in seconds
  geo: ""               # Geo coordinates as latitude,longitude
  public_key: ""        # If not provided, COMStation creates itself
tcpstation:
  enable: true
  address: "0.0.0.0:31313"  # IP and PORT to listen to, 0.0.0.0:31313 means available for everyone
  acl:                  # list of known addresses. If not specified accepts from everyone
httpstation:
  enable: true
  port: 8001
  # -
  # -
luftdaten:
  enable: true          # whether or not publish to https://luftdaten.info/
robonomics:
  enable: true          # enable use of Robonomics Network
  ipfs_provider: "/ip4/127.0.0.1/tcp/5001/http" # ipfs endpoint
  ipfs_topic: "airalab.lighthouse.5.robonomics.eth"
datalog:
  enable: false         # enable use of Datalog Robonomics subcommand
  path: ""              # path to Robonomics execution file
  suri: ""              # private key of publisher account
  remote: "wss://substrate.ipci.io"
  dump_interval: 3600   # time between two transactions in seconds
  temporal_username: "" # Temporal.Cloud credentials
  temporal_password: ""
dev:
  sentry: ""
